reviewcheck/app.py

```diff
@@ -243,10 +80,10 @@ def show_reviews(config: Dict[str, Any], no_notifications: bool) -> None:
     api_url = config["api_url"]
     jira_url = config["jira_url"]
     project_ids = config["project_ids"]
-    user = config["user"]
+    you = config["user"]
```

Comment:
I hate to comment on variable names but is `you` (and the `you_are_` pattern) really better than `user`?

Reply:
Good point... I wanted to differentiate between the reviewcheck user and the gitlab user. the gitlab user can be anyone, like any commenter in a thread. the reviewcheck user is the one you decided in config... maybe call it `gitlab_user` and `reviewcheck_user`? Or assume `user` is the user of reviewcheck since that is where the code is?

Reply:
I see, I think that's a fair assumption. Go with what you prefer though, just thought it was a question worth raising.

Reply:
Changed it

Reply:
As well as other variable/function names with "you" in them